### ISSUE TITLE
"Error: 'dict object' has no attribute u'WSL'" on all git commands through yadm

### ISSUE THREAD

**Issue by malikoth** [TheLocehiliosan/yadm#161]
### Describe the bug

I get an error every time I run a git command through yadm.  For instance, if I run `yadm remote`, here's what I see:
```
origin
Error: 'dict object' has no attribute u'WSL'
```

As you can see, the command _did_ execute, I got the output showing me the `origin` remote.  This becomes more of a problem, because one of my alt files is not generating through yadm, and it's failing with the same error message.  But not all of my alt files, the one that ends with `##WSL` actually generates just fine.  The alt file that I'm struggling with is my git config, which contains no Python, and no references to WSL at all.

I honestly have no idea if this is coming from YADM or not, and I can't even figure out how to find where this is coming from.  I see that detection for WSL was recently added, and indeed, I am seeing this in Ubuntu 18.04 on WSL.  Just figured I'd post this, and see if you have any insights or ideas for me to look into.  Thanks for your time, and for this wonderful project!!!

### To reproduce

Can this be reproduced with the yadm/testbed docker image: No

Long story short, I loaded up a container from that image, cloned my dotfiles repo, ran my bootstrap script, and ... it behaves as expected.

Steps to reproduce the behavior:  Only with my dotfiles repo, and only on WSL, run `yadm remote`

### Expected behavior

Only the remote `origin` is shown.

### Environment

 - Operating system: Ubuntu 18.04 on WSL
 - Version yadm: 1.12.0
 - Version Git: 2.22.0

### Additional context

[Add any other context about the problem here.]

**Comment by TheLocehiliosan** [comment]
@malikoth: That does look like an error message from Python to me. This is a guess, but I think it is related to this line in your Git configuration:

https://github.com/malikoth/dotfiles/blob/master/.config/git/config.managed%23%23yadm.j2#L19

My guess is **yadm** is attempting to to process your Jinja file, and the Jinja template itself is simply dereferencing a dictionary which doesn't have the WSL key in it. It looks like it only has the keys "Linux" and "Darwin". You probably need to add "WSL", or better yet, rewrite it so it won't choke on an unknown value.

**Comment by malikoth** [comment]
You found it!  I totally glossed over that line!  I knew I just needed another pair of eyes.  Thanks for your time, and have a grand day.
